draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1267,7 +1276,7 @@ established with {{-urireg}}.
 
 ### coap+ws

```

Comment:
I can convert the whole document easily in visual studio. You want Unix (LF) endings instead of (CR LF)?